### PR TITLE
fix(tab-title): add full focus outline to closable tab button in high contrast mode

### PR DESCRIPTION
**Pull request by mpriour**
Effect is only noticable in high-contrast mode. Issue #6994

**Related Issue:** #6994

## Summary
Adds a simple z-index to show the high contrast focus outline over the tab title outline.

### CODE REVIEW

File: packages/calcite-components/src/components/tab-title/tab-title.scss [hunk 1]
```diff
@@ -292,4 +292,8 @@
   :host([bordered][position="bottom"][selected]) .container {
     border-block-start-style: none;
   }
+
+  .close-button {
+    @apply z-default;
```
Comment (jcfranco) [inline packages/calcite-components/src/components/tab-title/tab-title.scss:297]: Could you add a comment explaining why the z-index is necessary in high contrast mode? 

@SkyeSeitz @ashetland Would `tab-title`'s close button end up at some point being circular (similar to `chip`)? Currently, the focus outlines from the title's container and close button mostly overlap and ended up causing some trouble in high contrast mode.

**Comment by jcfranco** [comment]
@mpriour Would you mind updating the PR title to indicate that this is specific to high-contrast mode?

**Review by jcfranco — APPROVED** [review]
Awesome! Thanks for helping out with this!

🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸
🎸🤘🎸🎸🎸🤘🎸🎸🤘🤘🎸🎸🤘🎸🎸🎸🤘🎸🎸🎸🤘🤘🤘🎸🎸🎸🤘🤘🎸🎸🎸🤘🤘🤘🎸🤘🎸🎸🤘🎸🤘🎸
🎸🎸🤘🎸🤘🎸🎸🤘🎸🎸🤘🎸🤘🎸🎸🎸🤘🎸🎸🎸🤘🎸🎸🤘🎸🤘🎸🎸🤘🎸🤘🎸🎸🎸🎸🤘🎸🤘🎸🎸🤘🎸
🎸🎸🎸🤘🎸🎸🎸🤘🎸🎸🤘🎸🤘🎸🎸🎸🤘🎸🎸🎸🤘🤘🤘🎸🎸🤘🎸🎸🤘🎸🤘🎸🎸🎸🎸🤘🤘🎸🎸🎸🤘🎸
🎸🎸🎸🤘🎸🎸🎸🤘🎸🎸🤘🎸🤘🎸🎸🎸🤘🎸🎸🎸🤘🎸🤘🎸🎸🤘🎸🎸🤘🎸🤘🎸🎸🎸🎸🤘🎸🤘🎸🎸🎸🎸
🎸🎸🎸🤘🎸🎸🎸🎸🤘🤘🎸🎸🎸🤘🤘🤘🎸🎸🎸🎸🤘🎸🎸🤘🎸🎸🤘🤘🎸🎸🎸🤘🤘🤘🎸🤘🎸🎸🤘🎸🤘🎸
🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸🎸

File: packages/calcite-components/src/components/tab-title/tab-title.scss (hunk 1 above)
Comment (ashetland) [reply packages/calcite-components/src/components/tab-title/tab-title.scss:297]: Yep, I think `tab-title` is a good candidate for the round button. We spec'd it out like the current design to match behaviors elsewhere until we could look at all these more holistically. #6640

Comment (ashetland) [reply packages/calcite-components/src/components/tab-title/tab-title.scss:297]: There might be padding tweaks needed to accommodate a round button.

Comment (mpriour) [reply packages/calcite-components/src/components/tab-title/tab-title.scss:298]: I've added the comment @jcfranco

**Pull request merged**